Campus-iOS/Base/Views/EmptyGradesView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TestView.swift
+//  Campus-iOS
+//
+//  Created by Mohanned Kandil on 23.10.2023.
+//
+
+
+import SwiftUI
+
+struct EmptyGradesView: View {
+    var body: some View {
+        GeometryReader { geo in
+            ZStack {
+                Image(decorative: "Error-logo-transparent")
```

Comment:
It's questionable for me if the error image is applicable here, since it's not an error